packages/storybook-html/stories/components/header/header.stories.js

```diff
@@ -32,45 +32,39 @@ export const HeaderInterim = {
     const headerElem = canvasElement.querySelector(".uq-header");
     if (headerElem) new header(headerElem);
 
-    document.addEventListener("DOMContentLoaded", function () {
-      const menuLeftElem = document.getElementById("global-mobile-nav");
-
-      const menuLeft = new SlideMenu(menuLeftElem, {
-        position: "left",
-        submenuLinkAfter: " ",
-        backLinkBefore: " ",
-      });
-
-      this.searchToggle = document.querySelector(".nav-primary__search-toggle");
+    const menuLeftElem = document.getElementById("global-mobile-nav");
```

Comment:
According to https://design-system.uq.edu.au/iframe.html?id=components-header--header&viewMode=story, it looks to me that `#global-mobile-nav` exists within `.uq-header` so you could do something like the below (to avoid querying the whole document again).

`const menuLeftElem = headerElem.getElementById("global-mobile-nav");`